com.ktechpit.colorwall.yaml

```diff
@@ -0,0 +1,88 @@
+app-id: com.ktechpit.colorwall
+runtime: org.kde.Platform
+runtime-version: 5.15-22.08
+sdk: org.kde.Sdk
+command: colorwall
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=xdg-download
+  - --filesystem=xdg-pictures
+  - --device=dri
+  - --env=PACKAGE_TYPE=flatpak
+  - --talk-name=org.freedesktop.FileManager1
+  - --filesystem=xdg-run/gvfsd
+  - --filesystem=~/.config/dconf:ro
+  - --talk-name=ca.desrt.dconf
+  - --env=DCONF_USER_CONFIG_DIR=.config/dconf
```

Comment:
these as well.

Reply:
did you try? gsettings works out of the box.

Reply:
I will test locally.

Reply:
Tried locally, the setting wallpaper with gsettings command is not working. It is changing the settings within the app snadbox but not affecting the user's background settings. What I may be missing?

Reply:
Apps like https://flathub.org/apps/im.bernard.Nostalgia use the wallpaper portal that would let to drop this sandbox hole. FWIW portals also work with Snap so it would be a win on both stores.